src/main/java/nl/tudelft/pixelperfect/client/CompleteMessage.java

```diff
@@ -0,0 +1,41 @@
+package nl.tudelft.pixelperfect.client;
+
+import com.jme3.network.serializing.Serializable;
+
+/**
+ * The message that the server recieves when an event is complete.
+ * 
+ * @author Dmitry
+ *
+ */
+@Serializable
+public class CompleteMessage {

```

Comment:
Actually, if you check my branch on GUI linking I made an EventCompletedMessage, which basically does the same.